core_data_modules/traced_data/traced_data.py

```diff
@@ -46,15 +46,16 @@ def get_call_location():
         """
         Returns the location where this function was called from.
 
-        :return Caller location in the format 'path/to/file.py:line:function_name'
+        :return Caller location in the format 'path/to/file.py:line_number:function_name'
         :rtype: str
         """
-        frame = inspect.stack()[1]  # Access the previous frame to find out where this function was called from.
-        f = frame[1]
-        line = frame[2]
-        name = frame[3]
+        # Access the previous frame to find out where this function was called from.
```

Comment:
```suggestion
        # Access the frame object for the caller’s stack frame to find out where this function was called from.
```
?

Reply:
Went with "Access the caller's stack frame" which I think is cleaner, thanks